iot-hub/Samples/device/PnpDeviceSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -101,6 +110,10 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
             _desiredPropertyUpdateCallbacks.Add(Thermostat1, TargetTemperatureUpdateCallbackAsync);
             _desiredPropertyUpdateCallbacks.Add(Thermostat2, TargetTemperatureUpdateCallbackAsync);
 
+            _logger.LogDebug("For each component, check if the device properties are empty on the initial startup.");
+            await CheckEmptyProperties(Thermostat1, cancellationToken);
```

Comment:
Correct suffix for async methods.

```suggestion
            await CheckEmptyPropertiesAsync(Thermostat1, cancellationToken);
```